packages/editor/src/components/rich-text/format-toolbar/index.js

```diff
@@ -11,7 +18,16 @@ const FormatToolbar = ( { controls } ) => {
 				{ controls.map( ( format ) =>
 					<Slot name={ `RichText.ToolbarControls.${ format }` } key={ format } />
 				) }
-				<Slot name="RichText.ToolbarControls" />
```

Comment:
I used this slot to add some toolbar controls.
There is no longer a way to add any controls at this position, isn't it?

Reply:
You should use the RichText API to add controls, not the slot id directly.

Reply:
Noted with thanks.